src/libtilt/projection/__init__.py

```diff
@@ -1,2 +1,2 @@
 from .project_fourier import project_fourier
-from .project_real import project_real
+from .project_real import project_real_3d, project_real_2d
```

Comment:
something about this doesn't feel 100% clear - how about `project_volume` and `project_image`? 🙂

Reply:
I changed it now to project_volume_real and project_image_real to not mess with potential variants from project_fourier. Let me know if you agree.